string filename
---
string command
float32 probability